Makefile

```
@@ -751,7 +751,7 @@ gen-api = tools/doc/generate.js --node-version=$(FULLVERSION) \
 		--versions-file=$(VERSIONS_DATA)
 gen-apilink = tools/doc/apilinks.js $(LINK_DATA) $(wildcard lib/*.js)
 
-$(LINK_DATA): $(wildcard lib/*.js) tools/doc/apilinks.js
+$(LINK_DATA): $(wildcard lib/*.js) tools/doc/apilinks.js | $(dir $(LINK_DATA))
```

```suggestion
$(LINK_DATA): $(wildcard lib/*.js) tools/doc/apilinks.js | out/doc
```
@DerekNonGeneric can you check if it works for you with this change please?

---

Yep, that worked!